src/Drupal/DrupalExtension/Context/DrupalContext.php

```diff
@@ -246,11 +270,8 @@ public function assertTextInTableRow($text, $rowText): void {
    * @Then I should see (the text ):text in all the :rowText rows
    */
   public function assertTextInTableRows($text, $rowText): void {
-    $rows = $this->getTableRows($this->getSession()->getPage(), $rowText);
-    foreach ($rows ?: [] as $row) {
-      if (strpos($row->getText(), $text) !== FALSE) {
+    if (!$this->hasTableRows($this->getSession()->getPage(), $rowText)) {
         throw new \RuntimeException(sprintf('Found a row containing "%s", but it did not contain the text "%s".', $rowText, $text));
```

Comment:
@kumikoono The indent should be:

```suggestion
    if (!$this->hasTableRows($this->getSession()->getPage(), $rowText)) {
      throw new \RuntimeException(sprintf('Found a row containing "%s", but it did not contain the text "%s".', $rowText, $text));
```